mozilla_version/test/test_version.py

```diff
@@ -158,6 +158,25 @@ def __init__(self, major_number):
 def test_base_version_substract_other_number_from_this_number(base_version, other, field, expected):
     assert base_version._substract_other_number_from_this_number(other, field) == expected
 
+@pytest.mark.parametrize('version_string, field, expected', (
+    ('0.9', 'major_number', '1.0'),
+    ('0.9.1', 'major_number', '1.0.0'),
+    ('32.0', 'major_number', '33.0'),
+    ('32.0.1', 'major_number', '33.0.0'),
+    ('32.0', 'minor_number', '32.1.0'),
```

Comment:
Hm, if `32.0` is considered valid and its bumped, why do we end up with `32.1.0`? Or since patch number is optional, `32.1.0` == `32.1` from `mozilla-version`'s perspective; the docstrings says `A new BaseVersion with the right field bumped and the following ones set to 0, if they exist.` but in this case, `patch_number` didn't exist in the first place.

Reply:
Ah good call. The documentation was ambiguous. I updated it. Tell me what you think!

Reply:
Hm, it still shows `A new {BaseVersion,GeckoVersion} with the right field bumped and the following ones set to 0, if they exist.` to me. Did you push the changes? 🤔